enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -37,6 +37,13 @@ def policy_pre_write_validation(policy_instance_or_class, field_values_by_name):
     If a constraint occurs, raises a `ValidationError`.
     """
     violations = []
+
+    if isinstance(policy_instance_or_class, SubsidyAccessPolicy):
+        policy_instance_or_class.clean()
```

Comment:
why would this if condition ever evaluate to false?

Reply:
I think the intent of this check is to only perform validation if `policy_instance_or_class` is an instance rather than a class.

```
>>> isinstance(AssignedLearnerCreditAccessPolicy, SubsidyAccessPolicy)
False
```